scripts/import/dbSNP_v2/update_old_sets.pl

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/perl
+
+## This script should get rs_names + variation_set_ids from last release
+## and "lift" all variation_id / variation_set_id to new dbSNP import.
+
+use strict;
+use DBI;
+use Bio::EnsEMBL::Registry;
+use Getopt::Long;
+
+## Connect to Databases
+
+my ($reg_file, $release, $tmp, $help);
+
+GetOptions ("registry=s"        => \$reg_file,
+            "release=s"      =>  \$release,
+            "tmp=s"         => \$tmp,
+            "help|h"           => \$help,
+);
+
+usage() unless defined $reg_file && defined $release && defined $tmp;
+
+my $registry = 'Bio::EnsEMBL::Registry';
+$registry->load_all($reg_file);
+
+my $db_adaptor = $registry->get_DBAdaptor("homo_sapiens", "variation");
+
+my $TMP_DIR = $tmp;
+my $TMP_FILE = "variation_feature.txt";
+
+my $tmp_vset_table = "variation_set_variation_e${release}";
+
+# define variants and move through the list
+my $dbh = $db_adaptor->dbc;
+my $old_release = $release - 1;
+my $old_dbname = $dbh->dbname =~ s/_${release}_/_${old_release}_/gr;
+my $old_host = $dbh->host;
+my $old_port = $dbh->port;
+
+my $old_dbh = DBI->connect("DBI:mysql:database=${old_dbname};host=${old_host};port=${old_port}", "ensro", "");
+
+## Stable variation_set.variation_set_id's values
+my @stable = qw(
+    3, 4, 5, 6, 7, 8, 9, 10, 
+    11, 12, 13, 14, 15, 16, 17, 
+    27, 40, 43, 44, 45, 47, 48, 
+    49, 50, 51, 52, 53, 54, 55, 56, 57
+  );
+
+### Get old rs_names / variation_set_ids
+## It will create a tmp file with variation_feature table values
+
+sub dumpPreparedSQL {
```

Comment:
It's easier to read the script if all the methods are at the end `dumpPreparedSQL` could be moved to line 201. 
This way we would have first the main code and then methods dumpPreparedSQL and usage.

Reply:
Done.